conf/include/tasks.yml

```diff
@@ -74,8 +74,6 @@ task: &sparse_parity_regression
 task: &retrieval
     <<: *task_base_config
     x_dim: 20
-    x_dist:
-        type: retrieval
```

Comment:
this will make the keys and values be sampled from standard gaussian, which is different than what the mambaformer paper did. 

<img width="1049" alt="task_info" src="https://github.com/in-context-learning-2024/in-context/assets/90002238/eabbbe04-38bf-4187-949d-39e167640672">

<img width="1049" alt="sampled" src="https://github.com/in-context-learning-2024/in-context/assets/90002238/81b489eb-da5d-45b5-a0c9-86117c7a3eca">



this is why there was this line of code in the original RetrievalDistribution:

`x_batch = x_batch / torch.norm(x_batch, p=2, dim=-1, keepdim=True) * math.sqrt(self.x_dim)`

Reply:
The code they have conflicts with this, normalizing to $\sqrt{d}$ rather than $1$. Further, the table claims that they use 32 k-v pairs, but they do no such thing. The sampler that supports a finite vocabulary size only works for x dimension 1!

Reply:
For consistency with their results (over their writing), I'll normalize to $\sqrt{d}$ and not limit ourselves to a finite vocabulary -- although this feels like the wrong way about this